<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerLobbyDialog</class>
 <widget class="QDialog" name="ServerLobbyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="BotRace::Core::GameSettingsWidget" name="settings" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Participants</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="participantListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Server Console</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="chatLogEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startStopGameButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopServerButton">
       <property name="text">
        <string>Stop Server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BotRace::Core::GameSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/gamesettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>settings</sender>
   <signal>valueChanged()</signal>
   <receiver>ServerLobbyDialog</receiver>
   <slot>changeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>startStopgame()</signal>
  <slot>stopServer()</slot>
  <slot>changeSettings()</slot>
 </slots>
</ui>
